packages/discord.js/src/structures/PollAnswer.js

```diff
@@ -75,14 +95,10 @@ class PollAnswer extends Base {
    * Fetches the users that voted for this answer.
    * @param {BaseFetchPollAnswerVotersOptions} [options={}] The options for fetching voters
    * @returns {Promise<Collection<Snowflake, User>>}
+   * @deprecated Use {@link PollAnswerVoterManager#fetch} instead
```

Comment:
Since this targets main, you can do a breaking change and drop the method, and it can be backported with a deprecation warning 👀

Reply:
This can be done in a follow-up PR. So this PR can be backported as-is